spec/api-menu-item-spec.js

```diff
@@ -389,4 +389,64 @@ describe('MenuItems', () => {
       expect(menu.items[0].submenu.items[0].overrideProperty).to.be.a('function')
     })
   })
+
+  describe('MenuItem accelerators', () => {
+    const isDarwin = () => {
+      return (process.platform === 'darwin')
+    }
+
+    it('should display modifiers correctly for simple keys', () => {
+      const menu = Menu.buildFromTemplate([
+        { label: 'text', accelerator: 'CmdOrCtrl+A' },
+        { label: 'text', accelerator: 'Shift+A' },
+        { label: 'text', accelerator: 'Alt+A' }
+      ])
+
+      assert.strictEqual(menu.getAcceleratorTextAt(0),
+        isDarwin() ? '⌘A' : 'Ctrl+A')
+      assert.strictEqual(menu.getAcceleratorTextAt(1),
+        isDarwin() ? '⇧A' : 'Shift+A')
+      assert.strictEqual(menu.getAcceleratorTextAt(2),
+        isDarwin() ? '⌥A' : 'Alt+A')
+    })
+
+    it('should display modifiers correctly for special keys', () => {
+      const menu = Menu.buildFromTemplate([
+        { label: 'text', accelerator: 'CmdOrCtrl+Tab' },
+        { label: 'text', accelerator: 'Shift+Tab' },
+        { label: 'text', accelerator: 'Alt+Tab' }
+      ])
+
+      assert.strictEqual(menu.getAcceleratorTextAt(0),
+        isDarwin() ? '⌘⇥\u0000' : 'Ctrl+Tab')
```

Comment:
where are these `\u0000` coming from? that doesn't seem right...

Reply:
that's where _you_ got them from, but not where they _came_ from :P

Reply:
@nornagon my best guess would be [that 0 at the end](https://cs.chromium.org/chromium/src/ui/base/accelerators/accelerator.cc?rcl=cc75805c1f32cad6601f75cf4cadfbee2344637d&l=296) that we (apparently) keep during conversion from `base::string16`